src/chars.ts

```diff
@@ -0,0 +1,5 @@
+// Charset from github.com/coreh/uid2
+const UIDCHARS = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789'
+
+export default UIDCHARS;
+module.exports = UIDCHARS;
```

Comment:
Why don't we just `export { UIDCHARS }`

Reply:
That was my first thought but I wanted to maintain backwards compatibility in case someone sneaking was importing this module directly.

Another way is to just move this variable into `index.ts` 😄